### ISSUE TITLE
How do I train and test using another dataset?

### ISSUE THREAD

**Issue by rabbit-pro** [LJOVO/TranSalNet#10]
Hi, Thank you for your code.

I'd like to ask, how do I train and test using another image dataset? Should I convert the fixation data into .bmp files?

**Comment by LJOVO** [comment]
Hi,

To train on different image datasets, it need to adjust the dataloader and training codes. 
It's not essential to convert fixation data to bmp files; the key is ensuring the NSS in the loss function can be calculated. 

Best
